content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+
+## Central Cluster Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you can deploy your services to the mesh, like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+From Istio 1.7, it is possible to run Istiod on a separate, dedicated cluster (the "central control plane cluster") as shown in the diagram below. With this new deployment model, mesh operators can work purely on the central control plane cluster when there is a need to upgrade or re-configure Istio. Platform/service owners work solely on the data plane cluster to deploy their applications and apply any Istio resources or configurations. They don't have direct access to the central control plane cluster to deploy anything there, neither can the central control plane cluster process any Istio resources in that cluster. If you are interested in exploring this, you can follow the [central istiod single cluster step by step guide](https://github.com/istio/istio/wiki/Central-Istiod-single-cluster-steps).
+
+{{< image width="100%"
+    link="single-cluster-central-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in a central control plane cluster"
+    >}}
+
+The diagram above shows a single cluster as the data plane for an Istio mesh. However, you can expand this deployment model to manage multiple Istio meshes from a central control plane cluster that runs multiple Istiod control planes:
+
+{{< image width="100%"
+    link="multiple-central-Istiods.svg"
+    alt="Istio meshes in single clusters with Istiod outside"
+    title="Multiple single clusters Istio meshes with Istiod outside"
+    caption="Multiple single clusters with multiple Istiod control planes in a central control plane cluster"
+    >}}
+
+Central control plane cluster can be used to host multiple Istiod control planes and each Istiod manages its own data plane clusters. In this model vendors could install another Istio mesh in the central control plane cluster and configure its `istio-ingress` gateway to route traffic from a data plane cluster to its corresponding Istiod control plane.
+
+You may further expand your data plane to multiple clusters, which are managed by the same Istiod running on the central control plane cluster. All of the data plane clusters will receive their configurations from the central control plane cluster per the diagram below.
```

Comment:
You should describe the difference between primary and remote cluster here, where primary just means that is where the config for the mesh comes from? Otherwise I don't understand why one is labelled primary and the other remote since they look identical.

Reply:
Thanks!  clarified - PTAL